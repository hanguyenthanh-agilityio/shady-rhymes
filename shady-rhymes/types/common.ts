export type NavItem = {
  label: string;
};

export type ListItems = {
  title: string;
  label: string[];
};

export type Service = {
  title: string;
  text: string;
  avatarName: string;
  avatarSrc: string;
  name: string;
  intro: string;
};

export type Product = {
  id: string;
  src: string;
  altText?: string;
  productName: string;
  helperText: string;
  price: string;
};
