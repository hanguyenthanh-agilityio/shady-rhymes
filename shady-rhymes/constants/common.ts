import { title } from 'process';
import { ListItems, NavItem, Product, Service } from '../types/common';

export const NAV_ITEMS: NavItem[] = [
  { label: 'Furniture' },
  { label: 'Shop' },
  { label: 'About us' },
  { label: 'Contact' }
];

export const INTRO = [
  {
    iconImage: '/icons/pro.svg',
    title: 'Professionalism',
    text: 'We deliver the final work with great professional way.',
    subText: 'more info'
  },
  {
    iconImage: '/icons/smooth.svg',
    title: 'Smooth workflow',
    text: 'We provide the most easy smoth workflow of interior design.',
    subText: 'more info'
  },
  {
    iconImage: '/icons/trusted.svg',
    title: 'Trusted Platform',
    text: 'People trust us since we openend the company',
    subText: 'more info'
  },
  {
    iconImage: '/icons/qualified.svg',
    title: 'Qualified employee',
    text: 'Our employees are the best of the field from the city',
    subText: 'more info'
  }
];

export const SERVICES: Service[] = [
  {
    title: 'Best planning service',
    text: 'I had fantastic experience at shady rhymes with pax planner, shady rhymes was so helpful and friendly and went above and beyond to help me configure my perfect walk in wardobe room.',
    avatarName: 'john',
    avatarSrc: 'https://bit.ly/dan-abramov',
    name: 'John Doe',
    intro: 'Project manager of cupalo'
  },
  {
    title: 'Best planning service',
    text: 'I had fantastic experience at shady rhymes with pax planner, shady rhymes was so helpful and friendly and went above and beyond to help me configure my perfect walk in wardobe room.',
    avatarName: 'john',
    avatarSrc: 'https://bit.ly/kent-c-dodds',
    name: 'John Doe',
    intro: 'Project manager of cupalo'
  },
  {
    title: 'Best planning service',
    text: 'I had fantastic experience at shady rhymes with pax planner, shady rhymes was so helpful and friendly and went above and beyond to help me configure my perfect walk in wardobe room.',
    avatarName: 'john',
    avatarSrc: 'https://bit.ly/code-beast',
    name: 'John Doe',
    intro: 'Project manager of cupalo'
  }
];

export const FEATURE: ListItems[] = [
  {
    title: 'Feature',
    label: ['Our Features', 'Pricing', 'Affiliate Program', 'Press Kit']
  },

  {
    title: 'Support',
    label: ['Account', 'Help', 'Contact Us']
  },

  {
    title: 'Legal',
    label: ['Privacy Policy', 'Terms of  Use']
  },

  {
    title: 'Media',
    label: ['Twitter', 'Product Hunt', 'Instagram', 'Github']
  }
];

export const PRODUCT: Product[] = [
  // {
  //   src: '/',
  //   altText: 'product',
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // },
  // {
  //   src: '/',
  //   altText: 'product',
  //   width: 400,
  //   height: 425,
  //   productName: 'Product',
  //   helperText: 'Category',
  //   subText: 'IDR 795.000',
  //   rating: 2
  // }
];
