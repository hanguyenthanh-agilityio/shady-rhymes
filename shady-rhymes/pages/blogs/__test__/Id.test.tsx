import { render } from '@testing-library/react';
import React from 'react';
import '@testing-library/jest-dom';
import { PRODUCT } from '../../../constants/common';

// Components
import Detail, { getStaticProps } from '../[id]';
import { ChakraProvider } from '@chakra-ui/react';

describe('Detail component', () => {
  it('Should render Detail snapshot correctly', () => {
    const detail = render(
      <ChakraProvider>
        <Detail
          blog={{
            id: '1',
            src: '/',
            altText: undefined,
            productName: '',
            helperText: '',
            price: ''
          }}
        />
      </ChakraProvider>
    );

    expect(detail).toMatchSnapshot();
  });

  // it('', async () => {
  //   global.fetch = jest.fn().mockImplementationOnce(() =>
  //     Promise.resolve({
  //       json: () => Promise.resolve({ data: PRODUCT })
  //     })
  //   );
  //   const res = await getStaticProps(1);
  // });
});
