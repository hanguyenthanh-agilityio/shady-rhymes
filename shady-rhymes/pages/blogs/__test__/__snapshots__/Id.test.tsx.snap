// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail component Should render Detail snapshot correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1lp9tiw"
      >
        <div
          class="chakra-stack css-1ujr5fn"
        >
          <div
            class="chakra-stack css-owjkmg"
          >
            <h2
              class="chakra-heading css-1xhp3dh"
            >
              Make your interior more minimaistic & modern
            </h2>
            <p
              class="chakra-text css-1o6irx4"
            >
              Bringing interiors to life, understanding the person of people living in a home is far greater need in design.
            </p>
          </div>
          <div
            class="chakra-stack css-ko33wr"
          >
            <div
              class="chakra-input__group css-1tv3gtv"
              data-group="true"
            >
              <input
                class="chakra-input css-1p61hk7"
                data-testid="search-input-field"
                name="search-input"
                placeholder="Search Furniture"
              />
              <div
                class="chakra-input__right-element css-dp9r5f"
              >
                <svg
                  class="chakra-icon chakra-icon css-1fyhvvq"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.414,20.591l-4.645-4.645a10.256,10.256,0,1,0-2.828,2.829l4.645,4.644a2.025,2.025,0,0,0,2.828,0A2,2,0,0,0,23.414,20.591ZM10.25,3.005A7.25,7.25,0,1,1,3,10.255,7.258,7.258,0,0,1,10.25,3.005Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="chakra-container css-mwn5ht"
      >
        <div
          class="css-i7o0my"
        >
          <div
            class="css-k008qs"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27425%27%20height=%27400%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                data-nimg="intrinsic"
                decoding="async"
                id="1"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="max-width: 100%; height: 0px; align-items: center; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; min-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(data:image/svg+xml;base64,Cjxzdmcgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgPGRlZnM+CiAgICA8bGluZWFyR3JhZGllbnQgaWQ9ImciPgogICAgICA8c3RvcCBzdG9wLWNvbG9yPSIjMzMzIiBvZmZzZXQ9IjIwJSIgLz4KICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzIyMiIgb2Zmc2V0PSI1MCUiIC8+CiAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMzMzMiIG9mZnNldD0iNzAlIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICA8L2RlZnM+CiAgPHJlY3Qgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIGZpbGw9IiMzMzMiIC8+CiAgPHJlY3QgaWQ9InIiIHdpZHRoPSI3MDAiIGhlaWdodD0iNDc1IiBmaWxsPSJ1cmwoI2cpIiAvPgogIDxhbmltYXRlIHhsaW5rOmhyZWY9IiNyIiBhdHRyaWJ1dGVOYW1lPSJ4IiBmcm9tPSItNzAwIiB0bz0iNzAwIiBkdXI9IjFzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgIC8+Cjwvc3ZnPg==);"
              />
              <noscript />
            </span>
          </div>
          <div
            class="chakra-stack css-1hadg1k"
          >
            <header
              class="css-0"
            >
              <h2
                class="chakra-heading css-0"
              />
              <p
                class="chakra-text css-9452eu"
              />
              <div
                class="css-k008qs"
              >
                <svg
                  color="#dfb300"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(223, 179, 0);"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                  />
                </svg>
                <svg
                  color="#dfb300"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(223, 179, 0);"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                  />
                </svg>
                <svg
                  color="#dfb300"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(223, 179, 0);"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                  />
                </svg>
                <svg
                  color="#dfb300"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(223, 179, 0);"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                  />
                </svg>
                <svg
                  color="#dfb300"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(223, 179, 0);"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                  />
                </svg>
              </div>
              <p
                class="chakra-text css-1bihoop"
              />
            </header>
            <div
              class="chakra-stack css-1aeb9ao"
            >
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-sl2ndi"
                >
                  Features
                </p>
              </div>
            </div>
            <button
              class="chakra-button css-122yr7x"
              type="button"
            >
              Add to cart
            </button>
          </div>
        </div>
      </div>
      <div
        class="css-vjhthh"
      >
        <div
          class="chakra-container css-0"
        >
          <div
            class="css-lsfylv"
          >
            <div
              class="chakra-stack css-1k6x2w"
            >
              <span
                style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 29.82456140350877% 0px 0px 0px;"
                />
                <img
                  alt="footer-logo"
                  data-nimg="responsive"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(data:image/svg+xml;base64,Cjxzdmcgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgPGRlZnM+CiAgICA8bGluZWFyR3JhZGllbnQgaWQ9ImciPgogICAgICA8c3RvcCBzdG9wLWNvbG9yPSIjMzMzIiBvZmZzZXQ9IjIwJSIgLz4KICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzIyMiIgb2Zmc2V0PSI1MCUiIC8+CiAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMzMzMiIG9mZnNldD0iNzAlIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICA8L2RlZnM+CiAgPHJlY3Qgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIGZpbGw9IiMzMzMiIC8+CiAgPHJlY3QgaWQ9InIiIHdpZHRoPSI3MDAiIGhlaWdodD0iNDc1IiBmaWxsPSJ1cmwoI2cpIiAvPgogIDxhbmltYXRlIHhsaW5rOmhyZWY9IiNyIiBhdHRyaWJ1dGVOYW1lPSJ4IiBmcm9tPSItNzAwIiB0bz0iNzAwIiBkdXI9IjFzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgIC8+Cjwvc3ZnPg==);"
                />
                <noscript />
              </span>
            </div>
            <div
              class="chakra-stack 1231134 css-1njlcjq"
            >
              <ul
                class="css-155za0w"
                role="list"
              >
                <p
                  class="chakra-text css-0"
                >
                  Feature
                </p>
                <li
                  class="css-kyfys8"
                >
                  <a
                    href="/"
                  >
                    Our Features
                    Pricing
                    Affiliate Program
                    Press Kit
                  </a>
                </li>
              </ul>
              <ul
                class="css-155za0w"
                role="list"
              >
                <p
                  class="chakra-text css-0"
                >
                  Support
                </p>
                <li
                  class="css-kyfys8"
                >
                  <a
                    href="/"
                  >
                    Account
                    Help
                    Contact Us
                  </a>
                </li>
              </ul>
              <ul
                class="css-155za0w"
                role="list"
              >
                <p
                  class="chakra-text css-0"
                >
                  Legal
                </p>
                <li
                  class="css-kyfys8"
                >
                  <a
                    href="/"
                  >
                    Privacy Policy
                    Terms of  Use
                  </a>
                </li>
              </ul>
              <ul
                class="css-155za0w"
                role="list"
              >
                <p
                  class="chakra-text css-0"
                >
                  Media
                </p>
                <li
                  class="css-kyfys8"
                >
                  <a
                    href="/"
                  >
                    Twitter
                    Product Hunt
                    Instagram
                    Github
                  </a>
                </li>
              </ul>
            </div>
          </div>
          <p
            class="chakra-text css-19fkxqh"
          >
            Copyright 2021 ©shady rhymes
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1lp9tiw"
    >
      <div
        class="chakra-stack css-1ujr5fn"
      >
        <div
          class="chakra-stack css-owjkmg"
        >
          <h2
            class="chakra-heading css-1xhp3dh"
          >
            Make your interior more minimaistic & modern
          </h2>
          <p
            class="chakra-text css-1o6irx4"
          >
            Bringing interiors to life, understanding the person of people living in a home is far greater need in design.
          </p>
        </div>
        <div
          class="chakra-stack css-ko33wr"
        >
          <div
            class="chakra-input__group css-1tv3gtv"
            data-group="true"
          >
            <input
              class="chakra-input css-1p61hk7"
              data-testid="search-input-field"
              name="search-input"
              placeholder="Search Furniture"
            />
            <div
              class="chakra-input__right-element css-dp9r5f"
            >
              <svg
                class="chakra-icon chakra-icon css-1fyhvvq"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M23.414,20.591l-4.645-4.645a10.256,10.256,0,1,0-2.828,2.829l4.645,4.644a2.025,2.025,0,0,0,2.828,0A2,2,0,0,0,23.414,20.591ZM10.25,3.005A7.25,7.25,0,1,1,3,10.255,7.258,7.258,0,0,1,10.25,3.005Z"
                  fill="currentColor"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="chakra-container css-mwn5ht"
    >
      <div
        class="css-i7o0my"
      >
        <div
          class="css-k008qs"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27425%27%20height=%27400%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              data-nimg="intrinsic"
              decoding="async"
              id="1"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="max-width: 100%; height: 0px; align-items: center; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; min-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(data:image/svg+xml;base64,Cjxzdmcgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgPGRlZnM+CiAgICA8bGluZWFyR3JhZGllbnQgaWQ9ImciPgogICAgICA8c3RvcCBzdG9wLWNvbG9yPSIjMzMzIiBvZmZzZXQ9IjIwJSIgLz4KICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzIyMiIgb2Zmc2V0PSI1MCUiIC8+CiAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMzMzMiIG9mZnNldD0iNzAlIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICA8L2RlZnM+CiAgPHJlY3Qgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIGZpbGw9IiMzMzMiIC8+CiAgPHJlY3QgaWQ9InIiIHdpZHRoPSI3MDAiIGhlaWdodD0iNDc1IiBmaWxsPSJ1cmwoI2cpIiAvPgogIDxhbmltYXRlIHhsaW5rOmhyZWY9IiNyIiBhdHRyaWJ1dGVOYW1lPSJ4IiBmcm9tPSItNzAwIiB0bz0iNzAwIiBkdXI9IjFzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgIC8+Cjwvc3ZnPg==);"
            />
            <noscript />
          </span>
        </div>
        <div
          class="chakra-stack css-1hadg1k"
        >
          <header
            class="css-0"
          >
            <h2
              class="chakra-heading css-0"
            />
            <p
              class="chakra-text css-9452eu"
            />
            <div
              class="css-k008qs"
            >
              <svg
                color="#dfb300"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                style="color: rgb(223, 179, 0);"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                />
              </svg>
              <svg
                color="#dfb300"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                style="color: rgb(223, 179, 0);"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                />
              </svg>
              <svg
                color="#dfb300"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                style="color: rgb(223, 179, 0);"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                />
              </svg>
              <svg
                color="#dfb300"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                style="color: rgb(223, 179, 0);"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                />
              </svg>
              <svg
                color="#dfb300"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                style="color: rgb(223, 179, 0);"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z"
                />
              </svg>
            </div>
            <p
              class="chakra-text css-1bihoop"
            />
          </header>
          <div
            class="chakra-stack css-1aeb9ao"
          >
            <div
              class="css-0"
            >
              <p
                class="chakra-text css-sl2ndi"
              >
                Features
              </p>
            </div>
          </div>
          <button
            class="chakra-button css-122yr7x"
            type="button"
          >
            Add to cart
          </button>
        </div>
      </div>
    </div>
    <div
      class="css-vjhthh"
    >
      <div
        class="chakra-container css-0"
      >
        <div
          class="css-lsfylv"
        >
          <div
            class="chakra-stack css-1k6x2w"
          >
            <span
              style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 29.82456140350877% 0px 0px 0px;"
              />
              <img
                alt="footer-logo"
                data-nimg="responsive"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(data:image/svg+xml;base64,Cjxzdmcgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgPGRlZnM+CiAgICA8bGluZWFyR3JhZGllbnQgaWQ9ImciPgogICAgICA8c3RvcCBzdG9wLWNvbG9yPSIjMzMzIiBvZmZzZXQ9IjIwJSIgLz4KICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzIyMiIgb2Zmc2V0PSI1MCUiIC8+CiAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiMzMzMiIG9mZnNldD0iNzAlIiAvPgogICAgPC9saW5lYXJHcmFkaWVudD4KICA8L2RlZnM+CiAgPHJlY3Qgd2lkdGg9IjcwMCIgaGVpZ2h0PSI0NzUiIGZpbGw9IiMzMzMiIC8+CiAgPHJlY3QgaWQ9InIiIHdpZHRoPSI3MDAiIGhlaWdodD0iNDc1IiBmaWxsPSJ1cmwoI2cpIiAvPgogIDxhbmltYXRlIHhsaW5rOmhyZWY9IiNyIiBhdHRyaWJ1dGVOYW1lPSJ4IiBmcm9tPSItNzAwIiB0bz0iNzAwIiBkdXI9IjFzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgIC8+Cjwvc3ZnPg==);"
              />
              <noscript />
            </span>
          </div>
          <div
            class="chakra-stack 1231134 css-1njlcjq"
          >
            <ul
              class="css-155za0w"
              role="list"
            >
              <p
                class="chakra-text css-0"
              >
                Feature
              </p>
              <li
                class="css-kyfys8"
              >
                <a
                  href="/"
                >
                  Our Features
                  Pricing
                  Affiliate Program
                  Press Kit
                </a>
              </li>
            </ul>
            <ul
              class="css-155za0w"
              role="list"
            >
              <p
                class="chakra-text css-0"
              >
                Support
              </p>
              <li
                class="css-kyfys8"
              >
                <a
                  href="/"
                >
                  Account
                  Help
                  Contact Us
                </a>
              </li>
            </ul>
            <ul
              class="css-155za0w"
              role="list"
            >
              <p
                class="chakra-text css-0"
              >
                Legal
              </p>
              <li
                class="css-kyfys8"
              >
                <a
                  href="/"
                >
                  Privacy Policy
                  Terms of  Use
                </a>
              </li>
            </ul>
            <ul
              class="css-155za0w"
              role="list"
            >
              <p
                class="chakra-text css-0"
              >
                Media
              </p>
              <li
                class="css-kyfys8"
              >
                <a
                  href="/"
                >
                  Twitter
                  Product Hunt
                  Instagram
                  Github
                </a>
              </li>
            </ul>
          </div>
        </div>
        <p
          class="chakra-text css-19fkxqh"
        >
          Copyright 2021 ©shady rhymes
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
